recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 228cf0748279546047eda25acb90b4fdaa344ad4d37b36e34c74d5558498172a
 
 build:
-  number: 0
+  number: 1
   script:
     - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
     - {{ PYTHON }} -m pip install . -vv
```

Comment:
Not sure why there's an error like `nothing provides _python_rc needed by python-3.12.0rc3-rc3_hab00c5b_1_cpython` (the conda solver shouldn't be trying to pull a release candidate version of Python), but could you try modifying L25 of this file to pin `maturin >=1.3,<1.4` following https://github.com/gauteh/hidefix/blob/v0.9.0/pyproject.toml#L2? Maybe that would help with the dependency solver.